recipe/meta.yaml

```diff
@@ -47,7 +47,7 @@ about:
     The Spatial Interaction Modeling (SpInt) module provides a collection of
     tools to study spatial interaction processes and analyze spatial
     interaction data.
-  doc_url: http://readthedocs.org/projects/spint
+  doc_url: https://github.com/pysal/spint
```

Comment:
```suggestion
  doc_url: https://spint.readthedocs.io/en/latest/index.html
```

no?

Reply:
dang it. have not had my coffee yet.

Reply:
And of course those docs [haven't been rebuilt since `v1.0.6`](https://spint.readthedocs.io/en/latest/?badge=latest)